src/abi/erctoken.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Technically this isn't really an ABI file so it's a bit weird. These are the function and event signatures and you can derive the ABI from this, but it's not an ABI json.

We might as well use the real ABI?

Reply:
And technically, `tokenAbi` and `erctoken` are roughly the same, I would just suggest we use the same ABI or only include the `approveAndCall()` on the `tokenAbi` instance.

Reply:
All the ABIs in this project are using the human-readable ABI format :smile: https://blog.ricmoo.com/human-readable-contract-abis-in-ethers-js-141902f4d917

I personally find it much easier to read and write manually. Do you think we should move everything back to the standard ABI format? I would rather use the same format for all the ABI files.

Reply:
Interesting... didn't know about this!

Although they are backwards compatible, it's still a bit weird to call them an ABI file because you wouldn't be able to put this into another tool expecting the "real" ABI json if it didn't also handle this format.

For sake of clarity, I would suggest moving back to the standard ABI format until this human readable format becomes more supported across the board (e.g. the [official ABI specification in the solidity docs](https://solidity.readthedocs.io/en/v0.6.2/abi-spec.html#json) adds it).